geotrek/signage/templates/signage/signage_detail_attributes.html

```diff
@@ -103,15 +110,22 @@ <h3>{% trans "Blades" %}</h3>
                         <td>{{ blade.type|default_if_none:"" }}</td>
                         <td>{{ blade.color|default_if_none:"" }}</td>
                         <td>{{ blade.direction }}</td>
-                        <td>{{ blade.condition|default_if_none:"" }}</td>
+                        <td>{% for condition in object.conditions.all %}
```

Comment:
```suggestion
                        <td>{% for condition in blade.conditions.all %}
```
This is displaying the condition of the signage, but it should be the blade's conditions here